### ISSUE TITLE
Which versions of R are supported?

### ISSUE THREAD

**Issue by gs-alt** [jpmml/jpmml-r#14]
For the supported model and transformation types, which versions of R are supported for this?

**Comment by vruusmann** [comment]
> which versions of R are supported for this?

The JPMML-R library should work fine with any R 3.X and 4.X version. Compatibility should be assessed on a R package basis, not R runtime basis.

The JPMML-R library interoperates with R package/runtime over "model dumps" in RDS data format. These dumps are fairly stable. When the data structure inside a model dump changes, then JPMML-R library tries to maintain compatibility with both old and new data structures.

TLDR: JPMML-R should be able to convert any model dump that has been produced in the last ~5 years. If you've discovered some incompatibilities, please open a new issue with more detailed description.

**Comment by gs-alt** [comment]
Thanks @vruusmann this is helpful
